P5/Source/Specs/cb.xml

```diff
@@ -8,7 +8,7 @@
   <gloss versionDate="2017-06-14" xml:lang="en">column beginning</gloss>
   <gloss versionDate="2007-12-20" xml:lang="ko">열 바꿈</gloss>
   <gloss versionDate="2007-05-02" xml:lang="zh-TW">分段</gloss>
-  <gloss versionDate="2009-01-06" xml:lang="fr">saut de colonne</gloss>
+  <gloss versionDate="2009-01-06" xml:lang="fr">début de colonne</gloss>
```

Comment:
when changing a description (or gloss) you must also update the corresponding `@versionDate` attribute (to the current date) on that element.